components/action-dropdown.js

```diff
@@ -8,7 +8,7 @@ export default function ActionDropdown ({ children }) {
   }
   return (
     <Dropdown className={`pointer ${styles.dropdown}`} as='span'>
-      <Dropdown.Toggle variant='success' as='a'>
+      <Dropdown.Toggle variant='success' as='a' onMouseDown={e => e.preventDefault()}>
```

Comment:
This is to prevent clearing a text selection when opening the action dropdown menu, thus enabling quoting selections.